docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -152,6 +152,10 @@ Elliptic Curve Signature Algorithms
 
 .. class:: EllipticCurvePublicNumbers(x, y, curve)
 
+    .. warning::
+        The point represented by this object is not validated in any way until
```

Comment:
"is not validated in any way until [...], it may not represent a valid point on the curve. You should not attempt to perform any computations using the values from this class, until you have either validated it yourself or called `public_key()` successfully"

WDYT?